client/components/jetpack/daily-backup-status/action-buttons.jsx

```diff
@@ -51,7 +53,7 @@ const RestoreButton = ( { disabled, rewindId } ) => {
 			isPrimary
 			className="daily-backup-status__restore-button"
 			href={ href }
-			disabled={ ! canRestore }
+			disabled={ isRestoreDisabled }
```

Comment:
I think we should make sure that the button isn't clickable by changing the `href` attribute as well? I know this is a problem with the `Button` component itself, but can't we fix it in general by adding `pointer-events: none` to the class responsible for the disabled state of the button?

Reply:
You can see just above that `href` is set to `undefined` if `isRestoreDisabled` is `true`.

Reply:
Thanks, I didn't notice that.